components/list-item/src/index.js

```diff
@@ -6,15 +6,16 @@ import {
   LINE_HEIGHT,
   MEDIA_QUERIES,
   NTA_LIGHT,
+  SPACING,
 } from '@govuk-react/constants';
 
 const GListItem = glamorous.li({
-  boxSizing: 'border-box',
   fontFamily: NTA_LIGHT,
   fontWeight: 400,
   textTransform: 'none',
   fontSize: FONT_SIZE.SIZE_14,
   lineHeight: LINE_HEIGHT.SIZE_14,
+  marginBottom: SPACING.SCALE_1,
```

Comment:
i think there's actually a media query associated with this in govuk-frontend - but it's not clear to me exactly how to read how that stuff works

Reply:
yeah - the actual CSS that's kicking in is: ```css
@media (min-width: 40.0625em)
.govuk-list > li, .govuk-prose-scope ul > li, .govuk-prose-scope ol > li {
    margin-bottom: 5px;
}
```